src/eureka/lib/utc_tt.py

```diff
@@ -70,7 +73,8 @@ def leapdates(rundir):
         jd = ls/86400+2415020.5
         return jd
     else:
-        print('NIST leap-second file not available.	Using stored table.')
+        log.writelog('  NIST leap-second file not available. '
+                     'Using stored table.')
 
         return np.array([2441316.5, 2441682.5, 2442047.5, 2442412.5,
```

Comment:
Since the leapsecond file is just downloaded automatically by the code (and I've now updated the saved table), I don't think it is worth explicitly providing a file within Eureka